tests/angular/custom-ignore-attributes/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+const {ignoreAttributes} = require('@ni/eslint-config-angular/options');
```

Comment:
This is a new public api so added a test for it. Updated the export to be from a generic `/options` entry point

Was on the fence about breaking up the options more, like should we

[1] export config for:
`@ni/eslint-config-angular` from `@ni/eslint-config-angular/options`
and
`@ni/eslint-config-angular/template` from `@ni/eslint-config-angular/template/options`?

[2] Or just export every option from: `@ni/eslint-config-angular/options` as this PR now currently does.

I think I kinda like [1] more which is an idea I had after pushing the change but wanted feedback first. @jattasNI

Reply:
I kinda like the symmetry of 1. If needed in the future we could also add '@ni/eslint-config-angular/requiring-type-checking/options'.

Reply:
Yea true. Lets go with [1]. I'll push a change for it.